.github/workflows/examples.yml

```diff
@@ -1,22 +1,38 @@
+# NOTE: This workflow seems to do nothing but install the library through conda
 name: Examples
 
-on: [push, pull_request]
+on:
+  # Manual trigger option in github
+  workflow_dispatch:
+
+  # Trigger on push to these branches
+  push:
+    branches:
+      - master
+      - development
+
+  # Trigger on a open/push to a PR targeting one of these branches
+  pull_request:
+    branches:
+      - master
+      - development
 
 jobs:
+
   ubuntu:
+    if: false  # This workflow aciton does nothing at the moment
 
-    runs-on: ubuntu-18.04
-    strategy:
-      matrix:
-        python-version: [3.9]
-      fail-fast:  false
+    runs-on: ubuntu-lateset
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Setup Python ${{ matrix.python-version }}
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Setup Python 3.9
       uses: actions/setup-python@v2
       with:
-        python-version: ${{ matrix.python-version }}
+        python-version: '3.9'
```

Comment:
should we always use the latest python version?

Reply:
and does the python version ref in line 23/39
   ```$CONDA/bin/conda create -n testenv --yes pip wheel gxx_linux-64 gcc_linux-64 swig python=${{ matrix.python-version }}```
need to be changed to ${{ python-version }} ?